clusterdock/models.py

```diff
@@ -365,6 +365,17 @@ def failure(timeout):
 
         self._add_node_to_etc_hosts()
 
+    def stop(self, remove=True):
+        """Stop the node by removing the Docker container.
```

Comment:
I think this should say `Stop the node and optionally removing the Docker container`.

Reply:
Done